.github/workflows/check.yaml

```
@@ -10,9 +10,11 @@ on:
       - ready_for_review
     branches:
       - main
+      - shinytest2@main
   push:
     branches:
       - main
+      - shinytest2@main
```

Make sure to revert the workflow before merging.
```suggestion
  push:
    branches:
      - main
```